pkg/flowaggregator/flowaggregator.go

```diff
@@ -489,7 +669,7 @@ func (fa *flowAggregator) sendFlowKeyRecord(key ipfixintermediate.FlowKey, recor
 		if err != nil {
 			return err
 		}
-		klog.V(4).Infof("Data set sent successfully: %d Bytes sent", sentBytes)
+		klog.V(4).InfoS("Data set sent successfully: %d Bytes sent", sentBytes)
```

Comment:
same comment as above. It should probably be:
```go
		klog.V(4).InfoS("Data set sent successfully", "bytes sent", sentBytes)

```

Reply:
Fixed. Thanks.